app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -112,9 +112,19 @@ function setCurrentUserToNavBar(user) {
   }
 }
 
+function setShieldIconVisibility(user){
+  const moderatorForTags = user.moderator_for_tags
+  if(user.admin || moderatorForTags.length > 0){
+    document
+      .getElementsByClassName('trusted-visible-block')[0]
+      .classList.remove('hidden');
```

Comment:
Using the suggested id (feel free to rename or use it differently though) and also [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in case it's not available for some reason (just a bit more safety IMO 😄 )

```suggestion
    document.getElementById('mod-shield-icon')?.classList.remove('hidden');
```

Reply:
Applied this change. Thanks